docs/cody/clients/install-vscode.mdx

```diff
@@ -130,9 +118,9 @@ function bubbleSort(array){
 
 ## Chat
```

Comment:
same comment as above, a disclaimer saying this is only available for enterprise would be great :)